src/renderer/shared/ui-entities/TransactionDetails/TransactionDetails.tsx

```diff
@@ -136,8 +136,6 @@ export const TransactionDetails = memo(({ wallets, chain, proxied, initiator, si
         </DetailRow>
       )}
 
-      {children ? <Separator className="border-filter-border" /> : null}
```

Comment:
let's keep this separator. it's used in all confirmation screens and this change can lead to large affected area.